### PR TITLE
updated composer.json

### PR DESCRIPTION
**Pull request by peshi**
allow symfony 2.8+

### CODE REVIEW

**Comment by peshi** [comment]
@iambrosi ping :)

**Comment by iambrosi** [comment]
@peshi Thanks for your PR. Can you also change the file [travis.yml](https://github.com/iambrosi/IsmaAmbrosiGeneratorBundle/blob/5d278d8b7a3ed39ff8d2e8853a570de6986ec781/.travis.yml#L19-L26) accordingly, so the new versions are also included in the tests?

**Comment by peshi** [comment]
@iambrosi yes ofc, do you want me to include php7 and hhvm?

**Comment by iambrosi** [comment]
I'd leave that for another PR, since it requires some tweaks to make the mongo extension work on those versions.

**Comment by peshi** [comment]
Hope this is OK

File: .travis.yml [hunk 1]
```diff
@@ -18,12 +17,12 @@ matrix:
   include:
     - php: 5.3
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
```
Comment (iambrosi) [inline .travis.yml]: What's the reason for removing PHP 5.3 and 5.4 from travis? versions 2.* need to work on PHP 5.3

File: .travis.yml [hunk 2]
```diff
@@ -31,6 +30,7 @@ before_install:
   - composer self-update
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
 
-install: composer update $COMPOSER_FLAGS --prefer-source
+install: composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
 
-script: ./vendor/bin/phpunit
+script:
+  - phpunit --coverage-text
```
Comment (iambrosi) [inline .travis.yml]: Coverage won't work, as xdebug [is disabled](#diff-354f30a63fb0907d4ad57269548329e3L29).

File: .travis.yml (hunk 1 above)
Comment (peshi) [inline .travis.yml]: Only reason I did that is this http://php.net/eol.php
And looking at many bundles 5.3 and 5.4 is removed from travis. I'll revert to its original state.

**Comment by peshi** [comment]
Let me know what you want changed, I'm not really the goto guy for travis things.

**Comment by iambrosi** [comment]
Thanks @peshi !!

**Pull request merged**